Mention Broot in Readme (#413)
trishume/syntect: Readme.md

- [tokio-cassandra](https://github.com/nhellwig/tokio-cassandra), CQL shell in Rust, uses `syntect` for shell colouring.
- [xi-editor](https://github.com/google/xi-editor), a text editor in Rust which uses `syntect` for highlighting.
- [Zola](https://github.com/getzola/zola), a static site generator that uses `syntect` for highlighting code snippets.
- [The Way](https://github.com/out-of-cheese-error/the-way), a code snippets manager for your terminal that uses `syntect`for highlighting.

> - [Broot](https://dystroy.org/broot), a terminal file manager, ises `syntect` for file previews

Reviewer: ```suggestion
- [Broot](https://dystroy.org/broot), a terminal file manager, uses `syntect` for file previews
```

> - [Broot](https://dystroy.org/broot), a terminal file manager, uses `syntect` for file previews

Reviewer 2: I would prefer us to follow the pattern and link directly to the github project

```suggestion
- [Broot](https://github.com/Canop/broot), a terminal file manager, uses `syntect` for file previews
```

Author: This makes sense as it helps people looking for snippets (there could maybe be a separate pages for the links to the relevant parts in the user codes, btw).

> - [Broot](https://github.com/Canop/broot), a terminal file manager, uses `syntect` for file previews

Reviewer 2: Sorry for not seeing this until now but all other bullets have a trailing `.` so let's add one to the new one too.

```suggestion
- [Broot](https://github.com/Canop/broot), a terminal file manager, uses `syntect` for file previews.
```
